scripts/community/achievements.js

```diff
@@ -927,6 +1080,10 @@ function InitAchievements( items, isPersonal )
 			gameLogoElement.hidden = true;
 
 			document.querySelector( '#topSummaryAchievements .achieveBar' )?.classList.add( 'steamdb_achievement_progressbar' );
+			if( isCompareView )
+			{
+				document.querySelector( '#topSummaryAchievements :nth-child(2 of .achieveBar)' )?.classList.add( 'steamdb_achievement_progressbar' );
```

Comment:
Just querySelectorAll( '#topSummaryAchievements .achieveBar' ) and add a class to them all?